src/views/DownloadPackedOrders.vue

```diff
@@ -59,8 +59,12 @@
             <ion-label>{{ fields[field] ? fields[field].label : field }}</ion-label>
           </ion-item>
         </ion-list>
-      </main>
-
+      </main> 
+      <main class="empty-state" v-else>
+        <img src="../assets/images/emptyState.png" />
```

Comment:
Rename file to `PackedOrderEmptyState.png`
```suggestion
        <img src="../assets/images/PackedOrderEmptyState.png" />
```